action/bucket.go

```diff
@@ -4,11 +4,20 @@ import (
 	"fmt"
 
 	log "github.com/sirupsen/logrus"
-	"github.com/yunify/qsctl/v2/task"
-
-	"github.com/yunify/qsctl/v2/contexts"
+	"github.com/yunify/qsctl/v2/storage"
+	"github.com/yunify/qsctl/v2/task/utils"
 )
 
+var stor storage.ObjectStorage
+
+func init() {
```

Comment:
init() does not work as expected, because we init config at the beginning of cmd run, but init() will exec at the first import, when the access key has not been parsed. So an "access key not provided" error was caused.